common/lib/azure/storage/common/client.rb

```diff
@@ -103,8 +103,11 @@ class << self
       # * +:default_endpoints_protocol+     - String. http or https
       # * +:use_path_style_uri+             - String. Whether use path style URI for specified endpoints
       # * +:ca_file+                        - String. File path of the CA file if having issue with SSL
+      # * +:ssl_version+                    - Symbol. The ssl version to be used
+      # * +:ssl_min_version+                - Symbol. The min ssl version supported
+      # * +:ssl_max_version+                - Symbol. The max ssl version supported
       # * +:user_agent_prefix+              - String. The user agent prefix that can identify the application calls the library
-      #
```

Comment:
Please add it back. It may break the document generation.

Reply:
added back.